### PR TITLE
[P5.js - hi] : Hindi translation for website copy

### PR DESCRIPTION
**Pull request by ranaaditya**
Translated following files into Hindi:
- src/content/ui/hi.yaml
- src/content/text-detail/hi/about.mdx
- src/content/text-detail/hi/index.mdx

### CODE REVIEW

File: src/content/text-detail/hi/about.mdx [hunk 1]
```diff
@@ -1,27 +1,27 @@
 ---
-title: "About"
+शीर्षक: "विषय में "
```
Comment (Divyansh013) [inline src/content/text-detail/hi/about.mdx:2]: I think "परिचय" will be better for about.

**Review by Divyansh013 — CHANGES_REQUESTED** [review]

File: src/content/text-detail/hi/about.mdx (hunk 1 above)
Comment (ranaaditya) [reply src/content/text-detail/hi/about.mdx]: done

**Review by Divyansh013 — APPROVED** [review]

**Comment by Divyansh013** [comment]
@TanviKumar PTAL

**Comment by Divyansh013** [comment]
We can merge this one @Qianqianye @limzykenneth

**Pull request merged**